gui/app/templates/interactive/effect-options/cooldown.html

```diff
@@ -0,0 +1,24 @@
+<div class="effect-setting-container">
+    <div class="effect-specific-title"><h4>Which buttons should I put on cooldown?</h4></div>
+    <div class="cooldown-group-buttons">
+        <ul class="cooldown-group-buttons-list">
+            <li ng-repeat="button in boardButtons">
+                <input ng-click="buttonArray(effect.buttons,button)" ng-checked="buttonCheckboxer(effect.buttons,button)" type="checkbox" aria-label="..." >
+                <span>{{button}}</span>
+            </li>
+        </ul>
+    </div>
+    <div class="cooldown-group-button-reset">
+        <button class="btn btn-warning">Uncheck all</button>
```

Comment:
So what we need to do is empty out the list of buttons that are checked.

```
$scope.uncheckAll = function() {
    $scope.effect.buttons = [];
}
```
Then add `ng-click="uncheckAll()"` to the button.